DRIVERS/ETH_16Z077/DRIVER/men_16z077_eth.c

```diff
@@ -3218,16 +3089,6 @@ static int chipset_init(struct net_device *dev, u32 first_init)
 				goto cont_init;
 			}
 		}
-#if LINUX_VERSION_CODE > KERNEL_VERSION(3,4,0)
```

Comment:
should this chunk of code be removed? It is only valid for kernel versions major to 3.4.0 so I think this is valid.